bin/node/src/aleph_cli.rs

```diff
@@ -93,8 +97,12 @@ impl AlephCli {
         self.enable_pruning
     }
 
-    pub fn alephbft_bit_rate_per_connection(&self) -> u64 {
-        self.alephbft_bit_rate_per_connection
+    pub fn alephbft_network_bit_rate(&self) -> u64 {
+        self.alephbft_network_bit_rate
+    }
+
+    pub fn substrate_network_bit_rate(&self) -> u64 {
```

Comment:
I'd prefer not to make this configurable (only a constant), as this is too low level for any typical node use case to be modified. Anyway, leaving this up to you, as alephbft is even more low level imho, and it's already there…

Reply:
Let's leave it as it is now.